imageroot/actions/clone-module/50Traefik

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import agent
+import os
+
+configure_retval = agent.tasks.run(agent_id=os.environ['AGENT_ID'], action='configure-module', data={
+    "lets_encrypt": os.environ["TRAEFIK_LETS_ENCRYPT"] == "True",
+    "host": os.environ["TRAEFIK_HOST"],
+})
```

Comment:
Sorry if I am wrong but TRAEFIK_HOST and TRAEFIK_LETS_ENCRYPT are not saved in environment but in the file config.json

you can read this file like [that](https://github.com/stephdl/ns8-pihole/blob/ec5161d36257602e6928c65fa522e2da1622b8e5/imageroot/actions/get-configuration/20read#L45)

Maybe you need to adapt following the format of the file